lib/order.rb

```diff
@@ -0,0 +1,68 @@
+# The Order class is a sub class of customer
+# it holds the dishes they choose to buy
+
+class Order
+
+  DEFAULT_TOTAL = 0 # Base cost of order
+
+  attr_accessor :order, :soter
+
+  def initialize
+    @order = []   # Initial array to recieve dishes
+    @soter  = []  # Holds refinned array of dishes and quantities
```

Comment:
Use of comments suggests that this code isn't clear. Are three arrays required for a single order?